Makefile.core.mk

```diff
@@ -6,6 +6,10 @@ test:
 build:
 	cargo build
 
+# Test that all important features build
+build-features:
+	cargo build --features console
```

Comment:
```suggestion
	RUSTFLAGS="${RUSTFLAGS} --cfg tokio_unstable" cargo build --features console
```

Reply:
I was trying to avoid this since it needs to rebuild everything again I think. Since we don't actually run it does it matter or do you think there are cases where the RUSTFLAGS may impact whether it builds?

Unfortunately we don't have caching of builds, so the build is fairly slow

Reply:
Yes, it would rebuild everything, which is unfortunate.

However, without `--cfg tokio_unstable`, the code in `tokio`, `tokio-tracing` and `tokio-console` that's guarded by `#[cfg(tokio_unstable)]` (and there is a lot of it) won't be compiled, so a check without it might result in some false positives.

I guess we could leave it as-is for now, add revisit if there is a false positive in the future.